todo-list-app/db/migrate/20170223142711_create_todos.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTodos < ActiveRecord::Migration[5.0]
+  def change
+    create_table :todos do |t|
+      t.string :item
+      t.string :content
+      t.boolean :is_completed
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
You only needed to add the code below which I see you included in your code. I'm curious to see how you use the item property.

```
 t.string :content
t.boolean :is_completed
```